src/app/profile/user-profile/header/header.component.html

```diff
@@ -0,0 +1,34 @@
+<header class="profile-header ">
+    <article fxLayout="row">
+        <img src="assets/stock/profile.jpg" class="profile-image" alt="Profile Picture">
+        <article class="user-details">
+            <h1>John Wick</h1>
+            <section class="ml-32 mt-8">
+                <h2>Hitman</h2>
+                <h3>Dontkillmydog@gmail.com</h3>
+                <h3><a href="www.linkedin.com" target="_blank">Personal Site</a></h3>
+            </section>
+        </article>
+    </article>
+    <article class="showcase-items">
+        <section fxLayout="column" fxLayoutAlign="center center">
+            <h3>Showcase Items</h3>
+            <h1 class="showcase mt-8" fxLayout="row" fxLayoutAlign="center center">8</h1>
+        </section>
```

Comment:
Maybe create a component out of that?

Reply:
Unless it won't be used anywhere else or it won't be more than 3

Reply:
to add to this, is a good practice to make a component for latter mantenability